templates/taxi/car_detail.html

```diff
@@ -5,8 +5,8 @@
   <h1>Drivers</h1>
   <hr>
   <ul>
-  {% for driver in car.drivers.all %}
-    <li>{{ driver.first_name }} {{ driver.last_name }}</li>
-  {% endfor %}
+    {% for driver in car.drivers.all %}
+      <li>{{ driver.first_name }} {{ driver.last_name }}</li>
```

Comment:
Ensure that each driver listed here is linked to their respective Driver detail page. You can achieve this by wrapping the driver's name in an anchor tag with the appropriate URL.